depparse/src/main/java/edu/illinois/cs/cogcomp/depparse/DepAnnotator.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * This software is released under the University of Illinois/Research and Academic Use License. See
+ * the LICENSE file in the root folder for details. Copyright (c) 2016
+ *
+ * Developed by: The Cognitive Computation Group University of Illinois at Urbana-Champaign
+ * http://cogcomp.cs.illinois.edu/
+ */
+package edu.illinois.cs.cogcomp.depparse;
+
+import edu.illinois.cs.cogcomp.annotation.Annotator;
+import edu.illinois.cs.cogcomp.annotation.AnnotatorException;
+import edu.illinois.cs.cogcomp.core.datastructures.Pair;
+import edu.illinois.cs.cogcomp.core.datastructures.ViewNames;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotation;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TreeView;
+import edu.illinois.cs.cogcomp.core.datastructures.trees.Tree;
+import edu.illinois.cs.cogcomp.core.io.IOUtils;
+import edu.illinois.cs.cogcomp.core.utilities.configuration.ResourceManager;
+import edu.illinois.cs.cogcomp.depparse.core.DepInst;
+import edu.illinois.cs.cogcomp.depparse.core.DepStruct;
+import edu.illinois.cs.cogcomp.depparse.core.LabeledChuLiuEdmondsDecoder;
+import edu.illinois.cs.cogcomp.sl.core.SLModel;
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+public class DepAnnotator extends Annotator {
+    private static Logger logger = LoggerFactory.getLogger(DepAnnotator.class);
+    private static SLModel model = null;
+    private static final String MODEL_NAME = "struct-perceptron-auto-20iter.model";
+    private static final String TEMP_MODEL_FILE_NAME = "tmp345673.model";
+
+    public DepAnnotator() {
+        super(ViewNames.DEPENDENCY, new String[] {ViewNames.POS, ViewNames.SHALLOW_PARSE,
+                ViewNames.LEMMA});
+    }
+
```

Comment:
We need a no-arg constructor that calls super() with a default resourcemanager object -- the specific Annotator constructor is:  
  public Annotator(String viewName, String[] requiredViews, boolean isLazilyInitialized,
            ResourceManager config) {